app-cdr/uif2iso/metadata.xml

```diff
@@ -1,9 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <longdescription lang="en">
```

Comment:
It's verbose, not very useful and technically incorrect: ISO9660 is a file system, not an archive file format, and ISO images might also contain other arbitrary file systems, for example UDF file system (https://en.wikipedia.org/wiki/ISO_9660 vs. https://en.wikipedia.org/wiki/ISO_image).